### PR TITLE
update meeting notes link to point to the new document

### PR DESCRIPTION
**Pull request by dprotaso**

### CODE REVIEW

File: working-groups/WORKING-GROUPS.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ The current working groups are:
 - [Security](#security)
   <!-- TODO add charters for each group -->
 
-## API Core
+## Serving API
```
Comment (psschwei) [inline working-groups/WORKING-GROUPS.md:69]: This breaks the section link above (L56), may want to update up there too

**Review by psschwei — CHANGES_REQUESTED** [review]

File: working-groups/WORKING-GROUPS.md (hunk 1 above)
Comment (psschwei) [reply working-groups/WORKING-GROUPS.md:69]: I'd also move this down so that the list stays in alphabetical order

Comment (dprotaso) [reply working-groups/WORKING-GROUPS.md:69]: oh nice catch

**Review by psschwei — APPROVED** [review]
/lgtm

**Pull request merged**